docs/en/connector-v2/source/kafka.md

```diff
@@ -59,6 +59,7 @@ They can be downloaded via install-plugin.sh or from the Maven central repositor
 ### Simple
 
 > This example reads the data of kafka's topic_1, topic_2, topic_3 and prints it to the client.And if you have not yet installed and deployed SeaTunnel, you need to follow the instructions in Install SeaTunnel to install and deploy SeaTunnel. And if you have not yet installed and deployed SeaTunnel, you need to follow the instructions in [Install SeaTunnel](../../start-v2/locally/deployment.md) to install and deploy SeaTunnel. And then follow the instructions in [Quick Start With SeaTunnel Engine](../../start-v2/locally/quick-start-seatunnel-engine.md) to run this job.
+> In batch mode, it will consume continuously until it reaches the maximum offset.
```

Comment:
Suggest move this hint to https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/kafka.md?plain=1#L14

And the `maximun` is not clear, in batch mode, it will stop when it reaches the offset at startup.

Reply:
This modification does not change any behavior of Kafka. I believe adding it above does not make a significant difference; it will consume up to the maximum offset of the shard

Reply:
This PR modification does not alter the behavior of the Kafka source. I believe adding it above does not make a significant difference and disrupts the structure of the documentation

Reply:
But I updated the description to explain the process